### PR TITLE
Tutorial: Use curly braces and `return` in ArticleCell Success

### PR DESCRIPTION
**Pull request by Tobbe**
Fixes #6205 #6214 

The scaffold generator produces cell components that use curly braces and a return statement. This PR updates the code snippet in the tutorial to match.

### CODE REVIEW

**Review by paybosllc — APPROVED** [review]

File: docs/docs/tutorial/chapter2/routing-params.md [hunk 1]
```diff
@@ -796,10 +796,10 @@ export const Failure = ({ error }: CellFailureProps) => (
   <div style={{ color: 'red' }}>Error: {error.message}</div>
 )
 
-export const Success = ({ article }: CellSuccessProps<ArticleQuery>) => (
+export const Success = ({ article }: CellSuccessProps<ArticleQuery>) => {
   // highlight-next-line
-  <Article article={article} />
-)
+  return <Article article={article} />
```
Comment (Philzen) [inline docs/docs/tutorial/chapter2/routing-params.md:801]: @Tobbe What about lines 764 & 766?

Comment (Tobbe) [reply docs/docs/tutorial/chapter2/routing-params.md:801]: Bah. JS. Who even cares? 😜
Just kidding. Of course those should be updated too.

Comment (Tobbe) [reply docs/docs/tutorial/chapter2/routing-params.md:801]: https://github.com/redwoodjs/redwood/pull/6235

Comment (Tobbe) [reply docs/docs/tutorial/chapter2/routing-params.md:801]: https://github.com/redwoodjs/redwood/pull/6235

**Pull request merged**